package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/hyperapp.js",
   "jsnext:main": "src/index.js",
   "module": "src/index.js",
+  "unpkg": "src/index.js",
```

Comment:
Using Hyperapp in the browser along to unpkg I would prefer to get the for-browser umd builded file `dist/hyperapp.js`.
The current behavior.

Reply:
I think I agree. But @JorgeBucaran recommended the `module` file [on Twitter the other day](https://twitter.com/JorgeBucaran/status/951009930609278976) so I was just following suit. It all depends on which file you'd expect to load when you use `https://unpkg.com/hyperapp`.